app/controllers/authentication_controller.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
 class AuthenticationController < ApplicationController
+  # Change this to localhost to sign in
+  REDIRECT_URI = 'https://gramgram.app/auth/instagram/callback'
```

Comment:
@edance would it be easier to make this an environment variable with dotenv?

Reply:
I was thinking the same thing. I was running out of gas last night though